src/components/Loader/Loader.styled.ts

```diff
@@ -28,5 +33,10 @@ export const StyledWrapper = styled.div`
   bottom: 0;
   width: 100%;
   background-color: ${(p) => p.theme.background.bg05};
+  opacity: 0.8;
   cursor: wait;
```

Comment:
Appreciation:- Was not aware about wait value for cursor property. Good that you have used it.